jmeter-director-worker/jmeter-director/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# This is the Dockerfile for the JMeter Director.
+# TODO: Refactor common code from JMeter Director / Worker into a Base file.
+
+FROM openjdk:slim
```

Comment:
we're trying to standardize on adoptopenjdk; could you use that instead? https://hub.docker.com/_/adoptopenjdk